### ISSUE TITLE
Deploy Option Removed From Workflowed Items In Umbraco Cloud

### ISSUE THREAD

**Issue by fnkymnky** [nathanwoulfe/Plumber#78]
**Describe the bug**
It seems the Deploy option is removed from Workflowed items for all user groups, including Admins, in Umbraco Cloud.

I'm 90% sure the option was available in a previous versions of Plumber.

**To Reproduce**
In all steps of the workflow process, the Deploy option doesn't appear in the save options panel for workflowed items. Here, you can see the option available for an item not under workflow:

![Options of an item not in workflow.](https://image.ibb.co/jCQVM9/without-workflow.png)

And here are all the options available at different stages of a workflow:

![Options of an item in workflow No.1](https://image.ibb.co/ibvpg9/with-workflow-1.png)
![Options of an item in workflow No.2](https://image.ibb.co/mRhKg9/with-workflow-2.png)

Notice no Deploy option.

**Expected behavior**
The deploy option provides a quick way to deploy content without having to queue it for transfer via the Actions dropdown and then navigate the Content tab. It should be available at all stages of the workflow process or at least have options to enable/disable it at various stages.

**Plumber version**
1.0.1

**Comment by nathanwoulfe** [comment]
This isn't something I've made any allowances for in previous versions, or changed in the current - I have a feeling the deploy button might have always been missing...

**Comment by fnkymnky** [comment]
Oh, really? I was sure I'd seen it before but perhaps it was on items that weren't under workflow.

Any plans to add it to the menu in a future release?

**Comment by nathanwoulfe** [comment]
Absolutely, it should be there. Not sure if it would be possible for the workflow to end with a deploy action though, but it should be in the menu for admins at least. 

Cloud seems to be adding a few different scenarios that are pretty unpredictable without an instance to test against...
